contrib/sacrebleu/sacrebleu.py

```diff
@@ -1228,12 +1271,14 @@ def main():
                             help='Read input from a file instead of STDIN')
     arg_parser.add_argument('refs', nargs='*', default=[],
                             help='optional list of references (for backwards-compatibility with older scripts)')
-    arg_parser.add_argument('--metrics', '-m', choices=['bleu', 'chrf'], nargs='+', default='bleu',
+    arg_parser.add_argument('--metrics', '-m', choices=['bleu', 'chrf'], nargs='+', default=['bleu'],
                             help='metrics to compute (default: bleu)')
     arg_parser.add_argument('--chrf-order', type=int, default=CHRF_ORDER,
                             help='chrf character order (default: %(default)s)')
-    arg_parser.add_argument('--chrf-beta', type=float, default=CHRF_BETA,
+    arg_parser.add_argument('--chrf-beta', type=int, default=CHRF_BETA,
                             help='chrf BETA parameter (default: %(default)s)')
+    arg_parser.add_argument('--chrf-whitespace', action='store_true', default=not CHRF_REMOVE_WS,
```

Comment:
Using 'store_true' together with a default dependent on a constant declared elsewhere is at least confusing and we could even end up with the default begin True.

Reply:
Are you suggesting we invert this? I want to default to excluding whitespace, but we could change the variable to `CHRF_ADD_WS` and invert the logic, if you think that's better and @fhieber signs off.

Reply:
I would remove the "default=..." and the CHRF_REMOVE_WS constant altogether.